File: Loesungen/loesung_05.cpp
// Bestimmt das kleinste gemeinsame Vielfachte (kgv)
// basic file operations
#include <iostream>
#include "library.h"
//#include "version.h"
using namespace std;


int main (int argc, char *argv[]) {
	int limite;

   std::cout << "Sexy Primzahlen mit 6 dazwischen" << std::endl;
   std::cout << "Grenze: ";
   std::cin >> limite;
   for (int i=1;i<limite;i++)
   {
	   if (isPrim(i) && isPrim(i+6)) {
	       std::cout << i <<  " ; " << i+6 <<std::endl;
	   }
   }
  return 0;
}

File: Loesungen/loesung_05.py
from library import *


if __name__ == "__main__":
    opt, args = getOpt()
    limite = int(args[0])
    for i in range(limite):
        #print(i,isPrim(i),isPrim(i+6))
        if isPrim(i) and isPrim(i+6):
            print("%d ; %d" % (i, i+6))
